manifests/consumer/config.pp

```diff
@@ -8,25 +8,25 @@
 # It manages the consumer config files
 #
 define kafka::consumer::config(
-  $config          = {},
+  $config          = $kafka::consumer::config,
+  $config_defaults = $kafka::consumer::config_defaults,
   $service_restart = $kafka::consumer::service_restart
 ) {
 
-  include ::kafka::params
-
-  $consumer_config = deep_merge($kafka::params::consumer_config_defaults, $config)
+  $consumer_config = deep_merge($config_defaults, $config)
 
   $config_notify = $service_restart ? {
-    true    => Service['kafka'],
+    true    => Service['kafka-consumer'],
     default => undef
   }
 
   file { "/opt/kafka/config/${name}.properties":
     ensure  => present,
```

Comment:
This is not necessary and we do not use it for our other modules. 
https://docs.puppetlabs.com/puppet/latest/reference/type.html#file-attribute-ensure